xknx/dpt/dpt_4byte_float.py

```diff
@@ -45,43 +45,331 @@ def to_knx(cls, value):
             raise ConversionError("Cant serialize %s" % cls.__name__, vlaue=value)
 
 
+class DPTAcceleration(DPT4ByteFloat):
+    """DPT 14,000 DPT_Value_Acceleration (ms-2)."""
```

Comment:
Please replace `DPT 14,*` with `DPT 14.*`. (these are parsed in https://github.com/XKNX/xknx/blob/master/home-assistant-plugin/home-assistant.io/sensor_table_generator.py )

Reply:
Will do.